pyproject.toml

```diff
@@ -1,19 +1,19 @@
 [build-system]
-requires = ["hatchling", "hatch-vcs", "sphinx-theme-builder @ https://github.com/pradyunsg/sphinx-theme-builder/archive/87214d0671c943992c05e3db01dca997e156e8d6.zip",]
+requires = ["sphinx-theme-builder @ https://github.com/pradyunsg/sphinx-theme-builder/archive/87214d0671c943992c05e3db01dca997e156e8d6.zip",]
```

Comment:
I played with using hatch vcs and not using it but it doesnt' make sense to have it because we are building with the theme-building tool not hatchling (i think??) i could be wrong.